lib/wifi_user/domain/sms_response.rb

```diff
@@ -0,0 +1,7 @@
+class WifiUser::Domain::SMSResponse
```

Comment:
I wouldn't introduce a domain object until it hits some level of complexity - I'd just pass around a hash, even from a gateway. 

I might introduce it where I know complexity is imminent, too - is this just the first slice of an `sms_response`?

Reply:
yes. The next piece of this is to add error handling.
I'd rather use a domain to define a static structure, even for something as simple as this.
Use of hashes very quickly leads to implicit complexity if left unchecked

Reply:
Cool, happy with this then.